atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/stream/StreamTest.java

```diff
@@ -50,11 +51,9 @@ public Long execute(Transaction t) throws Exception {
                 Sha256Hash hash = Sha256Hash.computeHash(data);
                 byte[] reference = "ref".getBytes();
                 long streamId = store.getByHashOrStoreStreamAndMarkAsUsed(t, hash, new ByteArrayInputStream(data), reference);
-//                store.markStreamsAsUsed(ImmutableMap.of(streamId, reference));
-//                store.removeStreamsAsUsed(ImmutableMap.of(1L, reference));
                 try {
                     store.loadStream(t, 1L).read(data, 0, data.length);
```

Comment:
was this supposed to be streamId (instead of 1L) and then not be in the try catch (to test loading)?